dashboard/app/controllers/levels_controller.rb

```diff
@@ -441,6 +442,8 @@ def new
         @game = Game.javalab
       elsif @type_class == Music
         @game = Game.music
+      elsif @type_class == Aichat
+        @game = Aichat.music
```

Comment:
This should probably be `@game = Game.aichat`.

Reply:
Good catch; I have no idea how I renamed it so badly!